types/jquery/legacy.d.ts

```diff
@@ -167,9 +167,9 @@ interface JQueryMouseEventObject extends JQueryInputEventObject {
  * @deprecated ​ Deprecated. Use \`{@link JQuery.Event }\`.
  */
 interface JQueryKeyEventObject extends JQueryInputEventObject {
-    char: any;
+    char: string;
     charCode: number;
-    key: any;
+    key: string;
     keyCode: number;
```

Comment:
Even though the interface itself is deprecated, it's probably still worth it to deprecate the individual properties too. I imagine there could be scenarios where tooling may not alert that a deprecated type is being used but would on a deprecated property.

Also, the deprecations serve different purposes. The one on the type encourages users to use the more accurate `JQuery.Event` type and only has compile-time consequences. The ones on the properties are to notify about potential loss of support and has run-time consequences.